examples/elytron-oidc-client/src/main/webapp/WEB-INF/oidc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "realm": "WildFly",
+  "auth-server-url": "${org.wildfly.bootable.jar.example.oidc.auth-server-url}",
+  "ssl-required": "external",
+  "resource": "simple-webapp",
```

Comment:
Instead of "realm" and "auth-server-url", we can use "provider-url" here instead (that way, it will be easier for anyone who wants to tweak this to use an OpenID provider other than Keycloak at some point).

Instead of "resource", we can use "client-id" here ("client-id" was introduced to use a name more similar to what the OIDC spec).

Reply:
@fjuma , thank-you, I aligned with OIDC spec.